.github/workflows/integ_test_logging.yml

```diff
@@ -64,6 +65,7 @@ jobs:
   logging-integration-test-watchOS:
     runs-on: macos-13
     environment: IntegrationTest
+    needs: [logging-integration-test-iOS, logging-integration-test-tvOS]
```

Comment:
Are the `needs` additions to ensure that the test suites on different platforms don't run in parallel?

Reply:
Is there a reasonable way to segment the log groups so that parallel runs can't interfere with each other?
I have two concerns:
1. Unnecessarily increasing our integration test run times in CI.
2. Tests manually triggered on a branch can break tests running during release process.